Stratis.Bitcoin.Tests/Utilities/PeriodicTaskTest.cs

```diff
@@ -46,8 +46,8 @@ public void StartWithoutDelayRunsTaskUntilCancelled()
 
             periodicTask.Start(new CancellationTokenSource(800).Token, TimeSpan.FromMilliseconds(300));
 
-            Thread.Sleep(1000);
-            Assert.Equal(3, this.iterationCount);
+            Thread.Sleep(2000);
+            Assert.True(this.iterationCount > 1);
```

Comment:
Ok yeah good for now, we need to eliminate this periodic task class (replace with AsyncLoop)